inst/examples/app_csv.R

```diff
@@ -0,0 +1,38 @@
+library(shiny)
+library(shiny.i18n)
+
+# file with translations
+n18i <- i18n(translation_csvs_path = "data")
```

Comment:
I'm not sure I like this name `n18i` --- feels hacky ;)

if we remove the function i18n, we can recommend in examples simply:

```
i18n <- Translator$new(....)
```

this seems more elegant to me, dont' you think?

Reply:
Yes